library/Vanilla/Embeds/TwitchEmbed.php

```diff
@@ -0,0 +1,209 @@
+<?php
+/**
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license https://opensource.org/licenses/GPL-2.0 GPL-2.0
+ */
+
+namespace Vanilla\Embeds;
+
+use Exception;
+
+/**
+ * Class for parsing twitch URLs, fetching twitch OEmbed data, and rendering server side HTML for a twitch video embed
+ */
+class TwitchEmbed extends VideoEmbed {
+
+    const DEFAULT_HEIGHT = 300;
```

Comment:
I know this seems redundant, given the name, but keep with the practice of documenting constants too.